File: CSCI-3800-Advanced-Programming/HW04/TateP_Problem4/Publisher.py
# Patrick Tate
# CSCI 3800 HW 4 Problem 4

# simple class to model a Publisher for a book
# only one member variable, a string name for the publisher
class Publisher:

    def __init__(self, name="None"):
        self.__name = name

    def getName(self):
        return self.__name

    def setName(self, name):
        self.__name = name

File: CSCI-3800-Advanced-Programming/HW04/TateP_Problem2/src/Publisher.java
/*
    - Patrick Tate
    - CSCI 3800 HW 4 Problem 2

 */

public class Publisher {
    private String name;

    public Publisher(String name){
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    /**
     * The toString method converts an object to a string. It is used to display the object in a println call.
     * @return
     */
    public String toString(){
        return name;
    }
}
